### ISSUE TITLE
Can't paste Word document

### ISSUE THREAD

**Issue by annechinn** [fraywing/textAngular#1361]
Our customers want to paste tables from their MS Word documents into textAngular, but many are having  problems. Attempt to paste it in either result in nothing being pasted or only some of the text without formatting.

Here is a link to one such document that fails to paste (I reproduced on the textAngular demo site).

https://drive.google.com/file/d/0BwaKQs36x9-pN2tmNHFtVERYVmM/view?usp=sharing

**Comment by JoelParke** [comment]
I have verified and beginning to take a look... 
Thanks for reporting this.

**Comment by JoelParke** [comment]
I have checked in a PR #1363 which corrects this. 

I will close this issue.  BUT I also noticed that much of the styling of font size, centering is done in MS with style='....' within the html which the sanitizer strips out as it must.   But it might be possible to do more to translate some of this into better formatting.  But this would be a fair bit of work and I am not sure that it is very helpful or necessary, since when the main structure has been pasted, it can then we styled within TA.  What are your thoughts?
